app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java

```diff
@@ -269,23 +279,22 @@ private void deleteFile(String name, int position) {
         final File fdelete = new File(name);
         mFileList.remove(position);
         notifyDataSetChanged();
-        getSnackbarwithAction(mActivity, R.string.snackbar_file_deleted)
-                .setAction(R.string.snackbar_undoAction, v -> {
-                    if (mFileList.size() == 0) {
-                        mEmptyStateChangeListener.setEmptyStateInvisible();
-                    }
-                    updateDataset();
-                    undoClicked.set(1);
-                }).addCallback(new Snackbar.Callback() {
-                    @Override
-                        public void onDismissed(Snackbar snackbar, int event) {
-                            if (undoClicked.get() == 0) {
-                            fdelete.delete();
-                            mDatabaseHelper.insertRecord(fdelete.getAbsolutePath(),
-                                    mActivity.getString(R.string.deleted));
-                            }
-                    }
-                }).show();
+        getSnackbarwithAction(mActivity, R.string.snackbar_file_deleted).setAction(R.string.snackbar_undoAction, v -> {
+            if (mFileList.size() == 0) {
```

Comment:
i dont see any change in this block of code, only spaces
please revet it

Reply:
because app:checkstyle fails and can't build the project